apps/studio/components/interfaces/Database/Extensions/ExtensionCard.tsx

```diff
@@ -36,30 +43,16 @@ const ExtensionCard = ({ extension }: ExtensionCardProps) => {
     setIsDisableModalOpen(true)
   }
 
-  async function disableExtension() {
-    try {
-      setLoading(true)
-      const response = await meta.extensions.del(extension.name)
-      if (!isResponseOk(response)) {
-        throw response.error
-      }
+  async function onConfirmDisable() {
```

Comment:
Doesn't need to be `async` anymore, right? 🔥
```suggestion
  function onConfirmDisable() {
```